internal/infrastructure/mongo/config.go

```diff
@@ -24,7 +25,7 @@ func NewConfig(client *mongodoc.Client) repo.Config {
 
 func (r *configRepo) Load(ctx context.Context) (*config.Config, error) {
 	cfg := &config.Config{}
-	if err := r.client.Collection().FindOne(ctx, nil).Decode(cfg); err != nil {
+	if err := r.client.Collection().FindOne(ctx, bson.D{}).Decode(cfg); err != nil {
```

Comment:
I don't think this change is necessary if config can be successfully loaded before saving it, why is this change necessary?